loops.js

```diff
@@ -32,7 +39,14 @@
 
 // Tip: een getal is oneven als je het door 2 deelt en er blijft nog iets over. Een getal is even als je het deelt door 2 en er blijft niets over. Google daarom eens naar 'mdn webdocs remainder operator'.
 // ==========================================
-
+const numbers = 0
+for (let numbers = 0; numbers < 16; numbers++){
+    if (numbers % 2){
```

Comment:
het lukt mij niet om dit te loggen, dus ik weet niet of het werkt zoals jij het nu schrijft.
Wat soweiso werkt is:   
if (numbers % 2 !== 0) {
  console.log(numbers + ' is oneven');